aws-py-oidc-provider/README.md

```diff
@@ -0,0 +1,126 @@
+# Provisioning an OIDC Provider in AWS for Pulumi Cloud
+
+WIP - This folder probably needs a better name to reflect that this example is configuring OIDC connection between Pulumi and AWS.
+
+This example is an automation of the process detailed in the [AWS documentation for creating an OIDC provider](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc.html). This automation will create OIDC configuration between Pulumi Cloud and AWS.
+
+## Prerequisites
+
+* [Install Pulumi](https://www.pulumi.com/docs/get-started/install/)
+* [Configure Pulumi to Use AWS](https://www.pulumi.com/docs/intro/cloud-providers/aws/setup/) (if your AWS CLI is configured, no further changes are required)
+
+## Running the Example
+
+Clone [the examples repo](https://github.com/pulumi/examples/tree/master/aws-py-oidc-provider) and navigate to the folder for this example.
+
+```bash
+git clone https://github.com/pulumi/examples.git
+cd examples/aws-py-oidc-provider-pulumi-cloud
+```
+
+Next, to deploy the application and its infrastructure, follow these steps:
+
+1. Create a new stack, which is an isolated deployment target for this example:
+
+    ```bash
+    $ pulumi stack init dev
+    ```
+
+1. Set your Pulumi organization name and desired AWS region:
+
+    ```bash
+    pulumi config set pulumiOrg <your-pulumi-org-name> # replace with your Pulumi organization name
+    pulumi config set aws:region us-east-1 # any valid AWS region will work
+    ```
+
+1. Install requirements.
+
+    ```bash
+    pip3 install -r requirements.txt
+    ```
+
+1. Run `pulumi up`. 
+
+    ```bash
```

Comment:
For maintenance reasons, I usually skip showing the CLI output. It gets stale and I don't think it adds that much for the reader. Your call.